tests/Spice86.Tests/MachineTest.cs

```diff
@@ -37,7 +37,7 @@ static MachineTest()
 
     public static IEnumerable<object[]> GetCfgCpuConfigurations()
     {
-        yield return new object[] { false };
+        // CfgCpu is now the only CPU implementation
```

Comment:
No, all tests should be updated to remove this parameter completely

Reply:
done.